mindmap/settings/common.py

```diff
@@ -10,4 +10,10 @@ def plugin_settings(settings):
     # AWS S3 settings
     settings.AWS_ACCESS_KEY_ID = "CHANGE-ME"
     settings.AWS_SECRET_ACCESS_KEY = "CHANGE-ME"
-    settings.FILE_UPLOAD_STORAGE_BUCKET_NAME = "CHANGE-ME"
+    settings.AWS_DEFAULT_ACL = None
```

Comment:
Be aware of: https://github.com/eduNEXT/xblock-mindmap/pull/8#issuecomment-1666500403